build/docker/zap-api-scan.py

```diff
@@ -197,26 +197,27 @@ def main(argv):
         usage()
         sys.exit(3)
 
+    if running_in_docker():
+        base_dir = '/zap/wrk/'
+        if config_file or generate or report_html or report_xml or progress_file or context_file or target_file:
+            # Check directory has been mounted
+            if not os.path.exists(base_dir):
+                logging.warning('A file based option has been specified but the directory \'/zap/wrk\' is not mounted ')
+                usage()
+                sys.exit(3)
+
     if target.startswith('http://') or target.startswith('https://'):
         target_url = target
     else:
         # assume its a file
-        if not os.path.exists(target):
+        if not os.path.exists(base_dir + target):
             logging.warning('Target must either start with \'http://\' or \'https://\' or be a local file')
```

Comment:
Thats because the user might have incorrectly missed off the https:// eg:
zap-api-scan.py -t www.example.com/openapi.json